_posts/2019-11-13-AWS-Dev-Day-2019.md

```diff
@@ -0,0 +1,76 @@
+
+## Conclusion
+Amplify seems like a really handy CLI tool if you enjoy working with AWS. The commands are very similar to each other, and really simplifies deploying your application or integrating your application with AWS components. A new feature that has been introduced to Amplify is Predictions, which provides a solution for Artificial Intelligence & Machine Learning cloud services to let your application make use of. Overall, the future for Amplify is very bright and I can see it growing more as more developers adopt it in their current workflow. After all, who doesn't like to just type one command and everything is taken care of?
+
+# Breaking the Monolith: road to containerizing your app
+A monolith is basically one big application that handles everything that your project needs to do. From security to having all your services in one package. This is simple, has no over-engineering (on the architecture part), faster communication as everything happens in-memory and you have one code base. BUT, monoliths grow old as well, and the older it gets, the more code it has. Your developers are also required to learn more business logic and your deployments are all or nothing, meaning that, if your application goes down, your whole infrastructure is down, which isn't interesting for an application where a lot of people depend on it.
+
+Dividing your monolith into a microservice architecture improves your workflow by splitting up your team to work on each microservice. This also helps your architecture in a way that all those microservices each have their own single responsibility. However, splitting a monolith is not a simple task and requires a lot of work to make sure you split it up the right way.
+
+Once you have your microservices, the goal is to containerize each service.
+
+## Fargate
+[Fargate](https://aws.amazon.com/fargate/){:target="_blank" rel="noopener noreferrer"} is a service that helps you with this. It's basically a serverless implementation for containers, meaning that it manages servers and clusters for you. Simply provision your container and Fargate does the rest for you by using Amazon ECS to provision your container on the cloud. It's a really useful tool if you don't want to manage your own server or cluster. 
+
+## Codebase, config, dependencies, ...
+The rest of the session was a very interesting take on how you actually split up your microservice. It's useful to have multiple codebases per service with their own deploy pipeline to deploy to your environments. 
```

Comment:
he rest of the session was a very interesting take on how you actually split up your monolith into microservices.